{
  "Header": {
    "Original": "CVE-2018-6596 (webhooks/base.py in Anymail (aka django-anymail) before 1.2.1 is prone ...)",
    "Line": 363503,
    "ID": "CVE-2018-6596",
    "Description": "(webhooks/base.py in Anymail (aka django-anymail) before 1.2.1 is prone ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4107-1}",
      "Line": 363504,
      "Type": "xref",
      "Bugs": [
        "DSA-4107-1"
      ]
    },
    {
      "Original": "- django-anymail 1.3-1 (bug #889450)",
      "Line": 363505,
      "Type": "package",
      "Package": "django-anymail",
      "Kind": "fixed",
      "Version": "1.3-1",
      "BugNo": 889450
    },
    {
      "Original": "NOTE: https://github.com/anymail/django-anymail/commit/db586ede1fbb41dce21310ea28ae15a1cf1286c5 (v1.3)",
      "Line": 363506,
      "Type": "NOTE",
      "Description": "https://github.com/anymail/django-anymail/commit/db586ede1fbb41dce21310ea28ae15a1cf1286c5 (v1.3)"
    },
    {
      "Original": "NOTE: https://github.com/anymail/django-anymail/commit/c07998304b4a31df4c61deddcb03d3607a04691b (v1.2.x-branch)",
      "Line": 363507,
      "Type": "NOTE",
      "Description": "https://github.com/anymail/django-anymail/commit/c07998304b4a31df4c61deddcb03d3607a04691b (v1.2.x-branch)"
    }
  ]
}